behat/symfony2-extension/2.1/behat.yaml.dist

```diff
@@ -0,0 +1,7 @@
+default:
+    extensions:
+        Behat\Symfony2Extension:
+            kernel:
+                bootstrap: vendor/autoload.php
```

Comment:
I tried this way: `@test -f vendor/bin/behat && vendor/bin/behat || echo "cannot run behat (needs behat/behat)"` with `kernel.bootstrap` set to `null`and all I'm getting is `Environment variable not found: "DATABASE_URL".`

it worked in this way: `@test -f vendor/bin/behat && DATABASE_URL=$(DATABASE_URL) vendor/bin/behat -v || echo "cannot run behat (needs behat/behat)"` and i see that Symfony Kernel is initialized with variables from behat configuration so APP_ENV and APP_DEBUG

I'll try to test it more complex